champagnepappi/WeConnect
# #155731891 add travis for automated testing

**What does this PR do**
- automates tests when the changes are made

**Description of tasks to be completed**
Automatically run test and install dependencies when change is committed.

**How should this method be tested**
clone the project
checkout to a feature  branch
write tests then push your changes
create a PR and you'll see the travis effect and be able to identify the failures in your test
